docs/en/fleet-developer/fleet-architecture.asciidoc

```diff
@@ -5,6 +5,10 @@ beta[]
 
 coming[a future update]
 
+## Elasticsarch as communication layer
```

Comment:
I recommend adding an ID to these headings so that you can link to them. Also add `[discrete]` tags before the headings to keep everything on one page. 

So for this section, you want something like:

```
[discrete]
[[fleet-communication-layer]]
== {es} as the communication layer
```

Reply:
added.